samples/beam/requirements.txt

```diff
@@ -1,3 +1,3 @@
 apache-beam==2.27.0
-google-cloud-bigtable==1.6.1
+google-cloud-bigtable==1.0.0
```

Comment:
this is an odd change. does 1.6.1 not work anymore? Do we need to range specifier (>= 1.6.1 < 2.0.0dev)?